src/deepsparse/transformers/pipelines/question_answering.py

```diff
@@ -501,7 +501,20 @@ def route_input_to_bucket(
         :param pipelines: Different buckets to be used
         :return: The correct Pipeline object (or Bucket) to route input to
         """
-        current_seq_len = len(input_schema.question.split())
+        tokenizer = pipelines[0].tokenizer
+        tokens = tokenizer(
```

Comment:
Because this code (afaict identical?) across transformers pipelines, it should be implemented on the transformers pipeline to be inherited by individual pipelines

Reply:
this is not identical (see the inputs to the tokenizer for QA here we need to join the context and question

Reply:
resolving